manifests/b/BraveSoftware/BraveBrowser/1.26.63/BraveSoftware.BraveBrowser.installer.yaml

```diff
@@ -0,0 +1,41 @@
+# Created using wingetcreate
+
+PackageIdentifier: BraveSoftware.BraveBrowser
+PackageVersion: 1.26.63

```

Comment:
```suggestion
PackageVersion: 91.1.25.73
```

Reply:
This is what shows up for me:
![image](https://user-images.githubusercontent.com/11600822/122421495-5003ef80-cf84-11eb-8fe1-c4da9aa4185a.png)

Looks like in the past couple of months, people weren't able to upgrade (other than doing `winget install BraveSoftware.BraveBrowser` to upgrade it) because the version number in the manifest is lower than the one shown in Control Panel.

Reply:
Ah I see, I'll sort that out for this PR.